.github/workflows/migration-tests.yml

```diff
@@ -0,0 +1,124 @@
+name: 'Migration Tests'
+
+on:
+  workflow_dispatch: {}
+  workflow_call: # workflow runs after code-health
+    inputs:
+      parent-event-name:
+        required: true
+        type: string
+  pull_request:
+    types: [ labeled ]
+
+jobs: 
+  set-output:
```

Comment:
@lantoli tried with another step: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1565/files#diff-9e63d3ed50a83cef3c77bb7447f2858374bc51e6dd94ac7f65b3420205c70ea6
This is an example PR running this workflow: https://github.com/maastha/terraform-provider-mongodbatlas/pull/1 

Let me know if this approach makes sense, not sure if this is possible with a variable

Reply:
that's great, thanks!